AwsCryptographicMaterialProviders/runtimes/java/src/examples/java/software/amazon/cryptography/example/hierarchy/CreateKeyExample.java

```diff
@@ -28,12 +31,34 @@ public class CreateKeyExample {
   public static String CreateKey(
     @Nonnull String kmsKeyArn,
     @Nullable String branchKeyId,
-    @Nullable KeyStoreAdmin admin
+    @Nullable KeyStoreAdmin admin,
+    @Nullable HierarchyVersion hierarchyVersion
   ) {
     // 1. Configure your Key Store Admin resource.
     final KeyStoreAdmin _admin = admin == null ? AdminProvider.admin() : admin;
+    final HierarchyVersion _hierarchyVersion = hierarchyVersion == null
+      ? HierarchyVersion.v1
+      : hierarchyVersion;
 
-    // 2. If you need to specify the Identifier for a Branch Key, you may.
+    // 2. Configure Key Management Strategy.
+    final KeyManagementStrategy strategy;
+    if (_hierarchyVersion == HierarchyVersion.v2) {
+      // Only KMS Simple is supported at this time for HV-2 to Create Keys
+      strategy =
+        KeyManagementStrategy
+          .builder()
+          .AwsKmsSimple(AwsKms.builder().build())
+          .build();
+    } else {
+      // Only KMS ReEncrypt is supported at this time for HV-1 Create Keys
+      strategy =
+        KeyManagementStrategy
+          .builder()
+          .AwsKmsReEncrypt(AwsKms.builder().build())
+          .build();
+    }
```

Comment:
Do these comments need to be updated, or does https://github.com/aws/aws-cryptographic-material-providers-library/blob/hv-2/hv-2/AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStoreAdmin/Model/KeyStoreAdmin.smithy#L246-L249?

Reply:
For public GA, we would like the Smithy Model's comments to be true.
For May 14th, we may live with what we have here.

I can take a TODO to make sure that these all align.